fluence-faas/src/faas.rs

```diff
@@ -184,6 +185,11 @@ impl FluenceFaaS {
         FaaSInterface { modules }
     }
 
+    /// Return statistic of Wasm modules heap footprint.
+    pub fn heap_statistic(&self) -> HeapStatistic<'_> {
```

Comment:
It's not a single value, so may be reflect that in the name? `heap_stats` would be nice. Or even `module_heap_stats`